src/calc.y

```diff
@@ -25,11 +25,17 @@ Term -> Result<Expr, ()>:
 Factor -> Result<Expr, ()>:
       '(' Expr ')' { $2 }
     | 'INT' { 
-        let s = $lexer.span_str($1.map_err(|_| "<evaluation aborted>").unwrap().span());
-        match s.parse::<u64>() {
-            Ok(n_val) => Ok(Expr::Number{ value: n_val }),
+        match $1.map_err(|err| format!("Parsing Error: {}", err)) {
+            Ok(s) => {
+              let s = $lexer.span_str(s.span());
+              match s.parse::<u64>() {
+                  Ok(n_val) => Ok(Expr::Number{ value: n_val }),
+                  Err(_) => Err(())
+              }
+            }
             Err(_) => Err(())
         }
+        
```

Comment:
I'll play laurie's role and say his catchphrase :) "blank line"

Reply:
You need to prefix this with "extraneous", otherwise you're just being a pedant :p

Reply:
blank line removed https://github.com/softdevteam/pavel.yaiwr/pull/8/commits/a251aa9643004316f6e7845f6a28d92712507134